src/main/java/uk/nhs/hee/tis/revalidation/connection/service/ExceptionElasticSearchService.java

```diff
@@ -47,7 +47,7 @@ public ExceptionSummaryDto searchForPage(String searchQuery, Pageable pageable)
       // add the free text query with a must to the column filters query
       BoolQueryBuilder fullQuery = mustBetweenDifferentColumnFilters.must(shouldQuery);
 
-      LOG.info("Query {}", fullQuery);
+//      LOG.info("Query {}", fullQuery);
```

Comment:
Maybe set this to debug, or even trace instead of getting rid of it completely?

Reply:
Updated to set it as debug. Cheers Andy.